src/gentropy/study_locus_validation.py

```diff
@@ -45,6 +45,8 @@ def __init__(
             .annotate_study_type(study_index)  # Add study type to study locus
             .qc_redundant_top_hits_from_PICS()  # Flagging top hits from studies with PICS summary statistics
             .qc_explained_by_SuSiE()  # Flagging credible sets in regions explained by SuSiE
+            # Flagging credible sets with PIP > 1 or PIP < 0.99
+            .qc_abnormal_pips(sum_pips_lower_threshold=0.99,sum_pips_upper_threshold=1)
```

Comment:
I guess that we are not dropping credible sets due to the lower bound, only the upper?

Reply:
Both below lower or above the upper bound are dropped, in theory - in practice I'm not so sure